phantom_connector.py

```diff
@@ -472,14 +478,15 @@ def _tag_artifact(self, param):
         # Post our changes
         ret_val, response, resp_data = self._make_rest_call(endpoint, action_result, data=art_data, method="post")
 
-        if (phantom.is_fail(ret_val)):
+        if phantom.is_fail(ret_val):
             self.save_progress("Unable to modify artifact")
             msg = PHANTOM_ERR_UPDATE_ARTIFACT.format(action_result.get_message())
             if not resp_label:
                 msg = "{}. {}".format("The reason of the failure can be the unavailability of the label in the provided artifact", msg)
             return action_result.set_status(phantom.APP_ERROR, msg)
 
-        action_result.set_summary({'tags_added': ', '.join((list(add_tags - tags_already_added))), 'tags_removed': ', '.join((list(remove_tags - tags_already_removed))),
+        action_result.set_summary({'tags_added': ', '.join((list(add_tags - tags_already_added))),
+        'tags_removed': ', '.join((list(remove_tags - tags_already_removed))),
          'tags_already_present': ', '.join((list(tags_already_added))), 'tags_already_absent': ', '.join((list(tags_already_removed)))})
```

Comment:
nit: Can we improve the indentation here? It looks inconsistent here, right?

Reply:
I have made the suggested code changes.